src/components/forms/DatePicker/DatePicker.test.tsx

```diff
@@ -0,0 +1,527 @@
+import React from 'react'
+import { render, act, fireEvent, createEvent } from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
+
+import { DatePicker } from './DatePicker'
+import { today } from './utils'
+import { DAY_OF_WEEK_LABELS, MONTH_LABELS } from './constants'
+
+/* 
+OUTSTANDING:
+- focus out on date picker external input validates
+- keydown on date picker input validates if keycode is enter
```

Comment:
Are these validation tasks still outstanding? If so lets file bugs for them.  I think one of these has a skipped test later on in the file, seems like we could remove it for now from here and add to the issue.